splunk_topology/check.py

```diff
@@ -105,8 +106,7 @@ def check(self, instance):
         except Exception as e:
             self._clear_topology(instance_key, clear_in_snapshot=True)
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, tags=instance.tags, message=str(e))
-            self.log.exception("Splunk topology exception: %s" % str(e))
```

Comment:
Was this too verbose? I'd like to see the error in the log aswell, or is the exception logged?

Reply:
No, the opposite was true. The trace was swallowed, the only thing we saw was the message which often was not clear enough. The trace included a trace up until this point. The fix is to include the original trace in an exception, hence `sys.exc_info`.

Reply:
nice